docs/_sources/documentation.rst.txt

```diff
@@ -87,18 +91,43 @@ To install AxonDeepSeg, you just need to install it with ``pip`` using the follo
   :: 
     pip install axondeepseg
 
+Option 2: Installing AxonDeepSeg in development mode (from GitHub)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. WARNING ::
+   Make sure that the virtual environment is activated before you run the following command.
+
+To install AxonDeepSeg in development mode, you first need to clone the AxonDeepSeg repository using the following command::
+
+    git clone https://github.com/neuropoly/axondeepseg.git
+
+Then, go to the newly created git repository and install the AxonDeepSeg package using the following commands::
+
+    cd axondeepseg
+    pip install -e .
 
 .. NOTE ::
-   In order to test the installation, you can launch an integrity test by running the following command on the terminal (make sure your virtual env is activated before, as explained in the `Creation a virtual environment <https://neuropoly.github.io/axondeepseg/documentation.html#creating-a-virtual-environment>`_ section) :
+   To update an already cloned AxonDeepSeg package, pull the latest version of the project from GitHub and re-install the application:
    ::
+    cd axondeepseg
+    git pull
+    pip install -e .
+
+Testing the installation
+-------------------------------------------------------------------------------
+
+In order to test the installation, you can launch an integrity test by running the following command on the terminal (make sure your virtual env is activated before, as explained in the `Creation a virtual environment <https://neuropoly.github.io/axondeepseg/documentation.html#creating-a-virtual-environment>`_ section)::
+
     axondeepseg_test
 
-   This integrity test automatically performs the axon and myelin segmentation of a test sample. If the test succeeds, the following message will appear in the terminal, meaning that the  software was correctly installed:
-   ::
-    * * * Integrity test passed. AxonDeepSeg is correctly installed. * * * 
 
+This integrity test automatically performs the axon and myelin segmentation of a test sample. If the test succeeds, the following message will appear in the terminal, meaning that the software was correctly installed::
+
+    * * * Integrity test passed. AxonDeepSeg is correctly installed. * * * 
 
 Models
+===============================================================================
+
+Existing models
```

Comment:
this header is irrelevant (because there is no header of the same level). i suggest to remove